OPCB/functions/missions/Secure.sqf

```diff
@@ -1,24 +1,18 @@
 private _reward = 60;
 params ["_base","_current_tasknumber"];
 _taskcomp = selectRandom ["warhead1","warhead2"];
-_guardgroup = createGroup [east,true];
 
 _current_task = _base getPos[random 600,random 360];
 [_current_tasknumber ,west,
 ["OPFOR is using the old abandoned FOB as a refueling point for transporting new technology through the region. Our mission is to intercept and secure the technology, preventing OPFOR from successfully delivering it and gaining a technological advantage.","Operation Phantom Grab"],
  _current_task,"ASSIGNED",10,true,true,"attack",true] call BIS_fnc_setTask;
 
-_guard = _guardgroup createUnit [OPCB_unitTypes_inf_commander, _base, [], 2, "NONE"];
-_guardpos = getPos _guard;
+_spawncomps = [_base] call CHAB_fnc_roadblock_rus;
+_comp = [_taskcomp,_base, [0,0,0], random 360, true, true ] call LARs_fnc_spawnComp;
 
-_spawncomps = [_guard] call CHAB_fnc_roadblock_rus;
-_comp = [_taskcomp,_guardpos, [0,0,0], random 360, true, true ] call LARs_fnc_spawnComp;
+[_base,east] call CHAB_fnc_enemySpawner;
 
-[_guard,10,1,3] call CHAB_fnc_spawn_rus;
-
-deleteVehicle _guard;
-
-_truckpos = _guardpos findEmptyPosition [1, 20, "O_Truck_03_device_F"];
+_truckpos = _base findEmptyPosition [1, 20, "O_Truck_03_device_F"];
```

Comment:
findSpot will clear and find an area that can hold the compound, then the vehicle can be safely place since the compound has a large enough space for the vehicle